applications/source/debezium-source/src/test/resources/docker/mssql/Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM mcr.microsoft.com/mssql/server:2022-latest
```

Comment:
Why can't we use Testcontainers for this? 

Also, there is some existing "sqlserver" directory under test/resources I believe.

Reply:
We need to instrument the SQL Server for [CDC](https://learn.microsoft.com/en-us/sql/relational-databases/track-changes/about-change-data-capture-sql-server?view=sql-server-ver16).  
Also, due to license rules we cannot distribute our own modified sql server image. Instead we have to build it on the fly. 

> Also, there is some existing "sqlserver" directory under test/resources I believe.
this old and will be removed.

Reply:
> Also, due to license rules we cannot distribute our own modified sql server image. Instead we have to build it on the fly.

I think we could use the `acceptLicense` API on the provided `MSSQLServerContainer` for this? 

It seems like we could do something like this instead:

```java
new MSSQLServerContainer("microsoft/mssql-server-linux:2017-CU9-GDR2")
				.acceptLicense()
				.withClasspathResourceMapping()
				.withCopyFileToContainer()
```
and get ride of the entrypoint and Dockerfile?